web/src/main/java/no/nav/ung/sak/web/app/tjenester/microsoftgraph/MicrosoftGraphRestKlient.java

```diff
@@ -0,0 +1,74 @@
+package no.nav.ung.sak.web.app.tjenester.microsoftgraph;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.core.UriBuilder;
+import no.nav.k9.felles.integrasjon.rest.ScopedRestIntegration;
+import no.nav.k9.felles.integrasjon.rest.SystemUserOidcRestClient;
+import no.nav.k9.felles.konfigurasjon.konfig.KonfigVerdi;
+import org.apache.http.Header;
+import org.apache.http.message.BasicHeader;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Optional;
+import java.util.Set;
+import java.util.regex.Pattern;
+
+@ApplicationScoped
+@ScopedRestIntegration(scopeKey = "microsofth.graph.scope", defaultScope = "https://graph.microsoft.com/.default")
+public class MicrosoftGraphRestKlient {
```

Comment:
Hadde det ikke vært bedre å bruke MS Graph SDK-en som har definert API for mulige kall? :)

Reply:
Jeg tenkte det var unødvendig å dra inn en ny avhengighet for å slippe å lage den lille User-klassen

Reply:
Jeg skjønner. Det hadde gjort fremtidige utvidelser noe enklere.

Hvis du ikke velger å bruke SDK-en hadde kanskje vært fint å lenke til API-dokumentasjonen i de relevante kallene :)

Reply:
Jepp, har lagt på peker til dokumentasjon